src/api/conceptApi.ts

```diff
@@ -59,3 +84,97 @@ export async function fetchConcepts(
     )
   ).filter(c => !!c);
 }
+
+export async function fetchDetailedConcept(
+  id: string,
+  context: Context,
+): Promise<GQLDetailedConcept> {
+  const response = await fetch(`/concept-api/v1/concepts/${id}`, context);
+  const concept: ConceptSearchResultJson = await resolveJson(response);
+  const detailedConcept: GQLDetailedConcept = {
+    title: concept.title.title,
+    content: concept.content.content,
+    subjectIds: concept.subjectIds,
+    copyright: concept.copyright,
+  };
+  let image;
+  const imageId = concept.metaImage?.url?.split('/').pop();
+  if (imageId) {
+    const imageResponse = await fetch(
+      `/image-api/v2/images/${imageId}`,
+      context,
+    );
+    image = await resolveJson(imageResponse);
+    detailedConcept.image = {
+      title: image.title.title,
+      src: image.imageUrl,
+      altText: image.alttext.alttext,
+      copyright: image.copyright,
+    };
+  }
+  if (concept.articleIds) {
+    const articlesResponse = await Promise.allSettled(
+      concept.articleIds.map(async articleId =>
```

Comment:
/articles har en parameter ids som tar inn en kommaseparert liste med ids. Du kunne kanskje brukt den for å spare nokre nettverkskall. Det er kanskje nok data i responsen der?

Reply:
Det gikk fint an